src/System.CommandLine.Suggest/dotnet-suggest.csproj

```diff
@@ -7,6 +7,7 @@
     <PackageId>dotnet-suggest</PackageId>
     <ToolCommandName>dotnet-suggest</ToolCommandName>
     <PackAsToolShimRuntimeIdentifiers>win-x64;win-x86;osx-x64;linux-x64</PackAsToolShimRuntimeIdentifiers>
+    <PackAsToolShimRuntimeIdentifiers Condition="'$(OSPlatformIsMusl)' == 'true'">linux-musl-x64</PackAsToolShimRuntimeIdentifiers>
```

Comment:
IMO we should just add `linux-musl-x64` to the list in the previous line.

Reply:
I'll try testing that - previously it had requested a slew of nupkgs, but it's been a whilte so I don't quite remember.